CHANGELOG.rst

```diff
@@ -43,6 +43,7 @@ Changed
 - feat: added auto generated feature table to readme, to track transform support for e.g. polars/lazyframes/json `#595 <https://github.com/azukds/tubular/issues/595>`_
 - feat: added FEATURE_REGISTRY dict maintained by BaseTransformer, which records per-transformer functionality info
 - feat: added `deprecated` class attr to deprecated classes, useful to filter them out of e.g. feature table
+- feat: turned on jsonable for  CappingTransformer `#637 <https://github.com/azukds/tubular/issues/637>`_
```

Comment:
Just for my own clarity, we will be adding the to_json method and updating the docstring for CappingTransformer with example of how to use to_json in a separate PR?